docs/discussion/maintenance-schedules.rst

```diff
@@ -54,27 +50,46 @@ we see fit over the following two weeks.
 
 On or around the expected release date, we will cut a final release.
 
+On Versioning
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+You may recognized from the note on the Northern Solstice release that ``ppb``
+uses a form of Calendar Versions or calver. Given a release version like
+``vy.m.p``:
```

Comment:
```suggestion
``y.m.p``:
```

Reply:
Mixed on this one, our version tags include the v usually, don't they?

Reply:
OH I see.

It depends on the exact semantics. Technically, the tag does, but it's not part of the number?

Reply:
Fair, I'll include the edit.